packages/hash/task-executor/requirements.txt

```diff
@@ -0,0 +1,5 @@
+certifi==2021.10.8
+charset-normalizer==2.0.9
+idna==3.3
+requests==2.27.1
+urllib3==1.26.7
```

Comment:
Are these just some "standard" libraries used in most python projects, or is there a reason why we have these currently?

Reply:
That's a good question, it's actually a remnant of a demo Python script I was writing before I replaced it with just returning a string. 

These are just `requests` and its sub-dependencies, and I included it to test an HTTP request (simulating that of querying a REST API of one of our integrations), I'll remove them from this PR though.